packages/shorebird_cli/lib/src/commands/patch/patch_ios_command.dart

```diff
@@ -543,7 +563,18 @@ ${summary.join('\n')}
         outputPath: _vmcodeOutputPath,
         workingDirectory: _buildDirectory,
         kernel: newestAppDill().path,
+        dumpDebugInfoPath: dumpDebugInfoDir?.path,
       );
+
+      if (dumpDebugInfo && dumpDebugInfoDir != null) {
```

Comment:
```suggestion
      if (dumpDebugInfoDir != null) {
```

Reply:
By adding the `dumpDebugInfoDir != null` I can skip the `unwrap` one line below.